schema_utils_test.py

```diff
@@ -98,6 +98,11 @@
             'type': SCHEMA_TYPE_UNICODE,
         },
     },
+    SCHEMA_TYPE_CUSTOM: {
```

Comment:
Why would all custom schemas have placeholders of type unicode? That seems like an overgeneralization.

Reply:
This line means that custom schemas may have placeholders of type unicode. A custom schema may not have such a placeholder. This can be seen [here](https://github.com/oppia/oppia/blob/develop/schema_utils_test.py#L172).

Reply:
That should be part of the schema only (i.e. part of the SanitizedUrl object) and not be made into a general thing.

Reply:
Done.